src/locales/zh_CN/battle.ts

```diff
@@ -73,6 +75,14 @@ export const battle: SimpleTranslationEntries = {
   "statHarshlyFell": "{{pokemonNameWithAffix}} 的 {{stats}} 大幅降低了！",
   "statSeverelyFell": "{{pokemonNameWithAffix}} 的 {{stats}} 极大幅降低了！",
   "statWontGoAnyLower": "{{pokemonNameWithAffix}} 的 {{stats}} 已经无法再降低了！",
+  "statRoseMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都提高了！",
+  "statSharplyRoseMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都大幅提高了！",
+  "statRoseDrasticallyMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都极大幅提高了！",
+  "statWontGoAnyHigherMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都已经无法再提高了！",
+  "statFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都降低了！",
+  "statHarshlyFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都大幅降低了！",
+  "statSeverelyFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都极大幅降低了！",
+  "statWontGoAnyLowerMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都已经无法再提高了！",
```

Comment:
```suggestion
  "statWontGoAnyLowerMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都已经无法再降低了！",
```
sorry my bad! wrong word here